simulation-ros/src/turtlebot2i/turtlebot2i_safety/testing_scripts/GenerateSceneGraph/README.md

```diff
@@ -0,0 +1,3 @@
+# This package is maintained by Shaolei Wang and Alberto Hata.
+
+- Enyu just copied the files.
```

Comment:
from?

Reply:
Sorry but I think @shaolei-wang doesn't upload this code. I got this code file from his computer. @albertohata Do you have some comments here?